common/src/float_ops.rs

```diff
@@ -226,7 +226,7 @@ pub fn to_string(value: f64) -> String {
             format!("{}e{:+#03}", significand, exponent)
         }
     } else {
-        value.to_string()
+        value.to_string().to_ascii_lowercase()
```

Comment:
You can do this in place without allocating a second string; `let mut  s = value.to_string(); s.make_ascii_lowercase(); s`